modules/infra/cli/notifications.c

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright (c) 2024 Christophe Fontaine
+
+#include <gr_api.h>
+#include <gr_cli.h>
+#include <gr_infra.h>
+#include <gr_net_types.h>
+
+static cmd_status_t notifications_dump(const struct gr_api_client *c, const struct ec_pnode *) {
+	struct gr_api_notification *n;
+	if (gr_api_client_enable_notifications(c) < 0)
+		return CMD_ERROR;
+
```

Comment:
There is no way to disable notifications without closing the connection.

It would be nice to handle `SIGINT` and send a `gr_api_client_disable_notifications()` to allow continue using the CLI.